unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUDDesktop/Configs/ResolutionControlConfiguration.asset

```diff
@@ -1,23 +1,23 @@
-%YAML 1.1
-%TAG !u! tag:unity3d.com,2011:
---- !u!114 &11400000
-MonoBehaviour:
-  m_ObjectHideFlags: 0
-  m_CorrespondingSourceObject: {fileID: 0}
-  m_PrefabInstance: {fileID: 0}
-  m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 0}
-  m_Enabled: 1
-  m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: 785002488258546488b783004eed1484, type: 3}
-  m_Name: ResolutionControlConfiguration
-  m_EditorClassIdentifier: 
-  title: Resolution
-  controlPrefab: {fileID: 9105552624476050456, guid: 0233968db98fb4cae83c48d54e190052,
-    type: 3}
-  controlController: {fileID: 11400000, guid: 2c861d5cb262b43cf99c14967d2925d6, type: 2}
-  flagsThatDisableMe:
-  - {fileID: 11400000, guid: e05eb73f83ed3453c80513ade0180f5a, type: 2}
-  flagsThatDeactivateMe: []
-  isBeta: 0
-  spinBoxLabels: []
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 785002488258546488b783004eed1484, type: 3}
+  m_Name: ResolutionControlConfiguration
+  m_EditorClassIdentifier: 
+  title: Resolution
+  controlPrefab: {fileID: 9105552624476050456, guid: 0233968db98fb4cae83c48d54e190052,
+    type: 3}
+  controlController: {fileID: 11400000, guid: 2c861d5cb262b43cf99c14967d2925d6, type: 2}
+  flagsThatDisableMe: []
```

Comment:
what was that Disable flag that you removed?

Reply:
The FullScreen borderless setting was disabling the resolution size change, we disabled this but somehow that disable flag was still there